.github/workflows/remove-label-on-approve.yml

```
@@ -1,26 +1,37 @@
-  pull_request_review:
+  pull_request_target:
     types:
-      - submitted
+      - opened
+      - synchronize
+      - reopened
 
 jobs:
   remove-labels-on-approve:
-    if: github.event.review.state == 'approved' # Trigger only when the review is approved
+    if: github.event.review?.state == 'approved' # Trigger only when the review is approved
```

actionlint (1.7.4)

got unexpected character '?' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)